docs/release.md

```diff
@@ -1,31 +1,86 @@
+     ```
+     For the following steps, use the remote configured for the `heptio/sonobuoy` repository.
+     The following instructions will use `upstream`.
+    * Push the tag with `git push upstream --tags` (note this will push all tags).
+      To push just one tag, use the following command format (replacing `v0.x.y` with the tag created in the previous step):
+      ```
+      git push upstream refs/tags/v0.x.y
+      ```
+      If there is a problem and you need to remove the tag, run the following commands:
+      ```
+      git tag -d v0.x.y
+      git push upstream refs/tags/v0.x.y
+      ```
+1. Open a browser tab and go to: https://travis-ci.org/heptio/sonobuoy/builds and verify go releaser for tag v0.x.y completes successfully.
+1. Upon successful completion of build job above, check the [releases tab of Sonobuoy](https://github.com/heptio/sonobuoy/releases) and verify the artifacts and changelog were published correctly.
+1. Run the [Jenkins job](https://jenkins.hepti.center/job/build-image-heptio-sonobuoy-release/build?delay=0sec) for pushing release images, specifying the release tag `v0.x.y` and confirm that the images get pushed correctly.
+2. Update the release notes if desired on GitHub by editing the newly created release.
 
 ## Validation
-1. Run the following command to make sure the image was pushed
-   correctly: `docker run -it gcr.io/heptio-images/sonobuoy:v0.x.y /sonobuoy version`. The output should
-   match the release tag above.  
-1. Go the the github release page (https://github.com/heptio/sonobuoy/releases) and download the release binaries and make sure the version matches the expected values.
-2. Run a Kind cluster locally and ensure that you can run `sonobuoy run --mode quick`. If this release corresponds to a new Kubernetes release as well, ensure:
-    - you're testing with the new Kind images
+1. Run the following command to make sure the image was pushed correctly:
+   ```
+   docker run -it gcr.io/heptio-images/sonobuoy:v0.x.y /sonobuoy version
+   ```
+   The `Sonobuoy Version` in the output should match the release tag above.
+1. Go to the [GitHub release page](https://github.com/heptio/sonobuoy/releases) and download the release binaries and make sure the version matches the expected values.
+2. Run a [Kind](https://github.com/kubernetes-sigs/kind) cluster locally and ensure that you can run `sonobuoy run --mode quick`.
+   If this release corresponds to a new Kubernetes release as well, ensure:
+    - you're testing with the new Kind images by checking the output from:
+      ```
+      export KUBECONFIG="$(kind get kubeconfig-path --name="kind")"
+      kubectl version --short
+      ```
+      and verify that the server version matches the intended Kubernetes version.
     - you can run `sonobuoy images` and get a list of test images as expected
 
+### Follow up
+Following the release when the new tag is made, the documenation will need to be updated to include the new version.
```

Comment:
I'll do the next PR which will clarify this and maybe add automation to getting those docs ready.

Reply:
Great, thanks!